Task2

andyskin opened the pull request:
RadioButton component is to be removed
All its functionality will be available in RadioGroup component

Commit:
Header is ready

Commit:
Other key elements of main page added

Commit:
Some responsive styles are added

Commit:
More responsive

Commit:
Additional components added

Commit:
Public folder isn't needed

Commit:
RadioBtn functionality is moved

Commit:
Started to work on movie info header

Commit:
Responsive styles are added to new header

Guria commented on client/Components/App/App.js:
```diff
@@ -0,0 +1,37 @@
+
+        return (
+            <div className="app">
+                <Header class={headerClass}>
```
зачем вводить путаницу? почему не использова className для миксов?

Guria commented on client/Components/App/App.js:
```diff
@@ -0,0 +1,37 @@
+        return (
+            <div className="app">
+                <Header class={headerClass}>
+                    {headerChildren}
```
```
page === "main"
  ? (
    <Header className="a">
      <Logo />
    </Header>
  ) : (
    <Header className="b">
      <Logo2 />
    </Header>
  ) 
```

Guria commented on client/Components/Button/Button.js:
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import styles from './button.css';
+
+export default class Button extends React.Component {
```
зачем класс?

Guria commented on client/Components/Footer/Footer.js:
```diff
@@ -0,0 +1,18 @@
+export default class Footer extends React.Component {
+    constructor(props) {
+        super(props);
+        this.footerClass = (props.class) ? `footer ${props.class}` : "footer";
```
не стоит так делать. вычисление класса лучше оставить в render и обойтись функциональным компонентом.  
для решения проблемы с классами посмотри на библиотеку classnames

Guria reviewed (commented):
Пока посмотрел поверхностно

Commit:
Classnames lib is added

Commit:
Code refactoring

App component is removed by const in index.js
Almost all classes have been rewritten to const
Started to work on routing

Commit:
Router and design changes

Commit:
Refactoring and Routing

Button is now button element instead of div
Routing is refactored
Header component is simplified

Commit:
Router is ready

Commit:
Refactoring

Added App component to simplify index.js, make routing settings more readable and avoid copypasting

Commit:
Renamed index component
